source/CommonJobs/CommonJobs.Mvc.UI/Web.config

```diff
@@ -17,6 +17,8 @@
     <add key="CommonJobs/DocumentationBaseUrl" value="http://commonjobs.makingsense.com/documentation" />
     <add key="LOGENTRIES_ACCOUNT_KEY" value="" />
     <add key="LOGENTRIES_LOCATION" value="" />
+    <add key="CommonJobs/GoogleOAuthClientId" value="293136734985.apps.googleusercontent.com" />
+    <add key="CommonJobs/GoogleOAuthSecret" value="z0IsQwpgFZBTRsmvSvzdThKV" />
```

Comment:
No, but it is not so important, it is my personal Google API secret, if someone stole it, the worst that could happen is that he consume all the quota, but it is really difficult. 
Of course if that happen, I will generate a new secret and change it outside code repository.